generator/plugins/python/utils.py

```diff
@@ -819,15 +837,20 @@ def _add_requests(self, lsp_mode: model.LSPModel) -> None:
         self._add_special("ResponseErrorMessage", ["error", "jsonrpc"])
 
         for request in lsp_mode.requests:
-            class_name = _to_class_name(request.method)
+            class_name = _get_class_name(request)
+            if not class_name.endswith("Request"):
+                class_name += "Request"
+            class_name_part = class_name.replace("Request", "")
+
             doc = _get_indented_documentation(request.documentation, indent)
 
+            params_class_name = f"{class_name_part}Params"
             if request.params:
                 if (
                     request.params.kind == "reference"
-                    and f"{class_name}Params" in CUSTOM_REQUEST_PARAMS_ALIASES
+                    and {params_class_name} in CUSTOM_REQUEST_PARAMS_ALIASES
```

Comment:
is the curly brackets what you want here? All the other ones for this line removed those, just wanted to check